### DISCUSSION TITLE
Attempting to build docker image with COPY --from on Actions

### DISCUSSION THREAD

**Discussion by WyriHaximus** [community/community#25599] [Actions]
<p>Attempting to port the creation of my public Docker images from CircleCI to Github Actions. Most of it seems to work but when it hits a</p>
<pre><code>COPY --from=build-meminfo /meminfo.so /meminfo.so
</code></pre>
<p> it errors out with: </p>
<pre><code>failed to export image: failed to create image: failed to get layer sha256:c6e65bd9400184963545d1bd9a6cc5b1d4ed71ff4974e646dd2f6e637eadd85d: layer does not exist
</code></pre>
<p>As can be seen: <a href="https://github.com/WyriHaximusNet/docker-php/runs/282412341#step:3:2693" rel="nofollow noopener">https://github.com/WyriHaximusNet/docker-php/runs/282412341#step:3:2693</a></p>
<p>As there a way around this limitation/bug?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi,</p>
<p>Please add an extra command (like RUN true) between the two COPY statements for a try.</p>
<p>In your Docker-zts(nts), there are 3 “copy --from” command, please add 2 ‘run true’ for each. I forked your repo, and it works for me.</p>
<p>You can refer to ticket below for the details, same error reported here:</p>
<p><a href="https://github.com/moby/moby/issues/37965" rel="nofollow noopener">https://github.com/moby/moby/issues/37965</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1fa5dd463dfdb11e108270a6e329e77800135dcc.png" data-download-href="/uploads/short-url/4vYabEoLbEfOy6fEintPApG3wGU.png?dl=1" title="Copy.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181091958-4b7c38ad-f79f-4842-85e7-283270a0c1b2.png" alt="Copy.png" data-base62-sha1="4vYabEoLbEfOy6fEintPApG3wGU" width="546" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1fa5dd463dfdb11e108270a6e329e77800135dcc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Copy.png</span><span class="informations">712×651 36.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by WyriHaximus** [comment]
<p>Thanks! That indeed does the trick, I have no clue why tho</p>

**Reply by utamas** [reply]
did that trick in our cases as well. Thank you!
